recipe/conda_build_config.yaml

```diff
@@ -70,27 +72,29 @@ flann:
 fontconfig:
   - 2.12
 freetype:
-  - 2.8                # [not ppc64le]
+  - 2.8.1              # [not ppc64le]
 gstreamer:
   - 1.12
 gst_plugins_base:
   - 1.12
+gdal:
+  - 2.2.*
 geos:
   - 3.6.2
 giflib:
   - 5.1
 glib:
-  - 2.53
+  - 2.55
 glpk:
-  - 4.62
+  - 4.61
```

Comment:
My reading of the [API/ABI report for glpk]( https://abi-laboratory.pro/tracker/timeline/glpk/ ) is that 4.61 and 4.62 are compatible, but 4.61 added one symbol after 4.60. Vaguely remember talking to @dougalsutherland about this some time ago.

Reply:
Yeah, I think that's right. We should maybe just update to the new 4.65 (which has new symbols), though.

Reply:
Could you please add that as a new issue? Would merely like to make sure we have copied everything over ok first. We can update them in a second pass.